terraform/api.tf

```diff
@@ -22,7 +22,8 @@ module "api" {
     "https://gui.dandiarchive.org",
     "https://gui-beta-dandiarchive-org.netlify.app",
     "https://codepen.io",
-    "https://cdpn.io"
+    "https://cdpn.io",
+    "https://bioimagesuiteweb.github.io",
```

Comment:
oh, this isn't a python, so may be trailing `,` would freak it out... let me remove